rse-cmdline-git.Rmd

```diff
@@ -0,0 +1,3536 @@
+# Git at the command line {#cmdline-git}
+
+## Questions {#cmdline-git-questions}
+
+```{r, child="questions/cmdline-git.md"}
+```
+
+## Objectives {#cmdline-git-objectives}
+
+```{r, child="objectives/cmdline-git.md"}
+```
+
+## Introduction {#cmdline-git-intro}
+
+A version control system stores a master copy of your code in a repository,
```

Comment:
Define "repository". These could be clarified depending on the target skill level, but I know most novices don't understand what the word "repository" means in the context of version control.